src/main/resources/beans.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
not necessary when using spring boot
configure in app.yml

Reply:
Newest commit implements your changes.